### DISCUSSION TITLE
is swarm required for multi server managment?

### DISCUSSION THREAD

**Discussion by j-mir-prazak** [Dokploy/dokploy#618] [Q&A]
Hi,
i did not find specific information about this in the docs, so i would like to ask: is it necessary to run swarm on all remote servers? i would like to use dokploy for managing project, however i would use different reverse proxy that i manage, however i am getting this:

"Error on  dockerfile / git Error: (HTTP code 503) unexpected - This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again."

which seems like the server is not connected to swarm so it wont run the deployment. this means that probably: the swarm should be init on the remote server setup - which it wasn't and there was no error; you need swarm even if you dont need the traefik proxy?

thank you for clarification!

best
jaromir

**Comment by Siumauricio** [comment]
Hello, yes you need docker swarm to run applications, according to what I read you didn't setup the server, when you run the setup it should automate everything without you having to do anything. 

Check the instructions here in order to setup a server succesfully
https://docs.dokploy.com/en/docs/core/multi-server/example#requirements
